core/ir/instr_shared.c

```diff
@@ -4196,4 +4206,4 @@ move_mm_avx512_reg_opcode(bool aligned64)
 }
 
 #endif /* !STANDALONE_DECODER */
-/****************************************************************************/
+       /****************************************************************************/
```

Comment:
Shouldn't be indented.

Reply:
clang-format is driving me crazy :) fixed.